test/models/MongoDateTimeFormatsSpec.scala

```diff
@@ -16,12 +16,13 @@
 
 package models
 
-import java.time.{LocalDate, LocalDateTime}
-
 import org.scalatest.{FreeSpec, MustMatchers, OptionValues}
 import play.api.libs.json.Json
+import uk.gov.hmrc.mongo.play.json.formats.MongoJavatimeFormats.Implicits._
+
+import java.time.{LocalDate, LocalDateTime}
 
-class MongoDateTimeFormatsSpec extends FreeSpec with MustMatchers with OptionValues with MongoDateTimeFormats {
+class MongoDateTimeFormatsSpec extends FreeSpec with MustMatchers with OptionValues {
```

Comment:
What is this testing now that MongoDateTimeFormats has been removed? I assume this can be deleted as we don't want to test the hmrc.mongo library function?

Reply:
you're right, this test is probably pointless now